app/views/bookings/index.html.erb

```diff
@@ -1,9 +1,5 @@
 <h1>Bookings#index</h1>
 
-<% if @bookings.nil? %>
-  <%= 'You have no bookings' %>
```

Comment:
On profile page I put "You have 0 bookings" and then in the else "You have @bookings.count" bookings".
Let's just decide what's better to make it homogeneous.